pennylane/tape/qscript.py

```diff
@@ -153,6 +155,15 @@ class QuantumScript:
     >>> qml.execute([qscript], dev, gradient_fn=None)
     [array([-0.77750694])]
 
+    Quantum scripts can also store information about the number and batches of
+    executions by setting the ``shots`` keyword argument. This information is internally
+    stored in a :class:`pennylane.measurements.Shots` object:
```

Comment:
```suggestion
    Quantum scripts can also store information about the number and batches of
    executions by setting the ``shots`` keyword argument:
```

Reply:
Probably don't need to tell users that we are using a `Shots` class if its an internal implementation detail and if the class is not user facing.

Reply:
By the fact of it getting returned, it is already exposed. And this class as a whole is pretty internal anyway.